faucet/src/Dscp/Faucet/Web/Server.hs

```diff
@@ -30,15 +31,21 @@ mkFaucetApiServer
 mkFaucetApiServer nat =
     hoistServer faucetAPI nat (toServant faucetApiHandlers)
 
+-- | CORS is enabled to ease development for frontend team.
+ourCors :: Middleware
```

Comment:
Why not `faucetCors` then? Two functions with the same name and different semantics is something my jump-to-definition won't understand :)

Reply:
Right, this doesn't make sense